### DISCUSSION TITLE
Anyone know when we can expect an updated Android library?

### DISCUSSION THREAD

**Discussion by keyanfayaz** [Azure/azure-iot-sdk-java#1494] [General]
Thinking about device/client implementations for Android devices. We are designing an IoT system and don't want to develop for old models. Appreciate the help/insight!

**Comment by timtay-microsoft** [comment]
We are actively exploring testing this SDK on newer Android API versions to see what the current bounds of the SDK are, and I'll update this thread when I have more findings from that research.

Is there a particular Android API version that you'd like to see this SDK supported on?

**Reply by AlanPintor-git** [reply]
All Android samples we find are for previous IoT versions (mostly for IoT Hub), we need one
SDK that works fine for the new IoT Central (APIv3)

Please let me know if you need more information. ty
